docs/getting-started.rst

```diff
@@ -161,74 +158,45 @@ Columns returned by the simulator
+| pflegev_beit_m          | Monthly amount of long term care insurance         | float |
++-------------------------+----------------------------------------------------+-------+
+| arbeitsl_geld_m         | Monthly amount of unemployment assistance          | float |
++-------------------------+----------------------------------------------------+-------+
+| rente_anspr_m           | Monthly amount of old-age pensions                 | float |
++-------------------------+----------------------------------------------------+-------+
+| abgelt_st               | Monthly capital income tax due, individual         | float |
++-------------------------+----------------------------------------------------+-------+
+| soli_st                 | Monthly solidarity surcharge due, individual       | float |
++-------------------------+----------------------------------------------------+-------+
+| kindergeld              | Monthly child Benefit, individual                  | float |
++-------------------------+----------------------------------------------------+-------+
+| eink_st                 | Monthly income Tax Due, individual                 | float |
++-------------------------+----------------------------------------------------+-------+
+| eink_st_tu              | Monthly income Tax Due, couple sum                 | float |
++-------------------------+----------------------------------------------------+-------+
+| unterhaltsvors_m        | Alimony advance payment, individual                | float |
++-------------------------+----------------------------------------------------+-------+
+| regelsatz_m             | Household socio-economic *need*, lump-sum          | float |
++-------------------------+----------------------------------------------------+-------+
+| kost_unterk_m           | Housing cost covered by social assistance          | float |
++-------------------------+----------------------------------------------------+-------+
+| kinderzuschlag_m        | Monthly additional child benefit, household sum    | float |
++-------------------------+----------------------------------------------------+-------+
+| wohngeld_m              | Monthly housing benefit, household sum             | float |
++-------------------------+----------------------------------------------------+-------+
+| arbeitsl_geld_2_m       | Monthly social assistance, household sum           | float |
++-------------------------+----------------------------------------------------+-------+
+| verfüg_eink_m           | Monthly disposable income, individual              | float |
++-------------------------+----------------------------------------------------+-------+
+| verfüg_eink_hh_m        | Monthly disposable income including household      | float |
+|                         | level benefits, household sum                      |       |
++-------------------------+----------------------------------------------------+-------+
```

Comment:
Don't we return `elterngeld` now? Should probably be `elterngeld_m`, too, by the argument above.

Please go through [all changes since 0.2.1](https://gettsim--156.org.readthedocs.build/en/156/changes.html#tbd) and check whether there are other cases where inputs/outputs require adjustment.

Reply:
Included `elterngeld_m`. I went through the changes. I think we caught all. No claim of completness though.